administrator/language/en-GB/plg_webservices_content.ini

```
@@ -5,3 +5,10 @@
 
 PLG_WEBSERVICES_CONTENT="Web Services - Content"
 PLG_WEBSERVICES_CONTENT_XML_DESCRIPTION="Add articles routes to the API for your website."
+PLG_WEBSERVICES_ALLOWEDVERBS_LABEL = "Allowed verbs"
+PLG_WEBSERVICES_FIELD_VALUE_GET = "GET"
+PLG_WEBSERVICES_FIELD_VALUE_POST = "POST"
+PLG_WEBSERVICES_FIELD_VALUE_DELETE = "DELETE"
+PLG_WEBSERVICES_FIELD_VALUE_PUT = "PUT"
+PLG_WEBSERVICES_FIELD_VALUE_PATCH = "PATCH"
+PLG_WEBSERVICES_PUBLIC_LABEL = "Allow public GET"
```

```suggestion
PLG_WEBSERVICES_PUBLIC_LABEL = "Allow Public GET"
```
Labels should be capitalized.